repository-data/application/src/main/resources/hcm-config/hst/configurations/common/sitemap.yaml

```diff
@@ -0,0 +1,13 @@
+definitions:
+  config:
+    /hst:hst/hst:configurations/common/hst:sitemap:
+      jcr:primaryType: hst:sitemap
+      /root:
+        jcr:primaryType: hst:sitemapitem
+        hst:componentconfigurationid: hst:pages/home
+        hst:relativecontentpath: publications/test-document
```

Comment:
this does not exist anymore :) but I guess we don't care to much as we don't have homepage anyway :D

Reply:
Indeed, no value in doing anything about this until we have specific requirements for the homepage.